### ISSUE TITLE
command-injection-process-builder (Java) false negatives

### ISSUE THREAD

**Issue by lfama** [returntocorp/sgrep-rules#1881]
**Describe the bug**
The rule doesn't match due to missing matching patterns for Windows OS (i.e. "cmd"), leading to false negatives.

**To Reproduce**
Run the rule against the following Java code:
```java
public class Test {
StringBuilder getResponseFromPingCommand(String ipAddress, boolean isValid) throws IOException {
        boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
        StringBuilder stringBuilder = new StringBuilder();
        if (isValid) {
            Process process;
            if (!isWindows) {
                process =
                        // Rule matches here
                        new ProcessBuilder(new String[] {"sh", "-c", "ping -c 2 " + ipAddress})
                                .redirectErrorStream(true)
                                .start();
            } else {
                process =
                        // But doesn't match here
                        new ProcessBuilder(new String[] {"cmd", "/c", "ping -n 2 " + ipAddress})
                                .redirectErrorStream(true)
                                .start();
            }
            try (BufferedReader bufferedReader =
                    new BufferedReader(new InputStreamReader(process.getInputStream()))) {
                bufferedReader.lines().forEach(val -> stringBuilder.append(val).append("\n"));
            }
        }
        return stringBuilder;
    }
}
```

**Expected behavior**
The rule should also match Windows "cmd" command.

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me

**Comment by 0xDC0DE** [comment]
Rule has been updated!
